src/components/MediaSettings/MediaSettings.vue

```diff
@@ -448,7 +464,7 @@ export default {
 			if (this.isInCall) {
 				localMediaModel.disableVirtualBackground()
 			} else {
-				BrowserStorage.setItem('virtualBackgroundEnabled_' + this.token, 'false')
+				BrowserStorage.removeItem('virtualBackgroundEnabled_' + this.token)
```

Comment:
Please also check BrowserStorage (devtools - application - local storage):
- url, type and enabled flag are stored by tokens for each conversation
- they appear on screen in Media Settings
- but option in the component points to "None", not the defined before image